keras_cv/layers/preprocessing/random_cut_mix.py

```diff
@@ -0,0 +1,126 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+from tensorflow.keras import layers
+
+
+class RandomCutMix(layers.Layer):
```

Comment:
Please add a class dosctring with `Args:` description and a usage example

Reply:
Done, added to both classes